src/token/api/src/error.rs

```diff
@@ -0,0 +1,51 @@
+use candid::{CandidType, Deserialize};
+use ic_helpers::tokens::Tokens128;
+use thiserror::Error;
+
+// TODO: a wrapper over `ic_helpers::TxError`, this is a most likely
```

Comment:
Since there's `ic_helpers::TxError` why do we need this one? 🤔

Reply:
I have never noticed that **TODO** , no we don't have `TxError` in ic-helpers